hdbt.theme

```diff
@@ -846,6 +847,53 @@ function hdbt_preprocess_paragraph(array &$variables) {
       $variables['icon_name'] = ucfirst($paragraph->get('field_icon')->icon);
     }
   }
+
+  // Map paragraph.
+  if (
+    $paragraph_type == 'map' &&
+    $paragraph->hasField('field_map_map') &&
+    $paragraph->hasField('field_iframe_title')
+    ) {
```

Comment:
Fix the indent, so that the closing bracket is on same indent as the starting if statement. It's easier to read if there's a lot of code. 
Like so: 
```
  if (
    $paragraph_type == 'map' &&
    $paragraph->hasField('field_map_map') &&
    $paragraph->hasField('field_iframe_title')
  ) {
```